frontend/src/pages/ArtistOverview.js

```diff
@@ -16,6 +17,14 @@ export default function ArtistOverview(){
             .catch((error) => console.error(error))
     },[artistName]);
 
+    if (artistName === "not-found"){
```

Comment:
I wondered about this in the backend already, this is not really robust, imagine an artist names `not-found`, better throw a 404 and handle it accordingly in the frontend.

Reply:
I see that this is not optimal and you have a point there is really an artist in my api who is called "not found" ;) I didnt find an example in github where an error is thrown and the user is forwarded to get a message about the issue. I changed the string to "error-not-found"...

Reply:
Haha, that's funny :) But the general issue remains with `error-not-found`, when you respond with a 404 from the backend, axios will throw and you can check the status of the axios response in the catch blog.